sources/packages/backend/workflow/test/2021-2022/parttime-assessment/costs/parttime-assessment-2021-2022-additional-transportation-costs.e2e-spec.ts

```diff
@@ -0,0 +1,391 @@
+import { PROGRAM_YEAR } from "../../constants/program-year.constants";
+import {
+  createFakeConsolidatedPartTimeData,
+  executePartTimeAssessmentForProgramYear,
+} from "../../../test-utils";
+import { OfferingDeliveryOptions, YesNoOptions } from "@sims/test-utils";
+const additionalTransportationOnsiteDistanceLimit = 280;
```

Comment:
IMO, this can be moved to the program year constants. It would copied over for every new program year and if it changes someday it can be adjusted. Also this would be case to it be declared as `ADDITIONAL_TRANSPORTATION_ONSITE_DISTANCE_LIMIT`.
https://github.com/bcgov/SIMS/blob/0554472731358a64d44688a966c799f963d01a41/sources/packages/backend/workflow/test/2021-2022/constants/program-year.constants.ts